certs.jsonnet

```diff
@@ -57,11 +57,14 @@
     "eclipse-pass.org": [
       "eclipse-pass.org",
       "www.eclipse-pass.org",
+      "demo.eclipse-pass.org",
+      "nightly.eclipse-pass.org",
     ],
 
     "eclipsepass.org": [
       "eclipsepass.org",
-      "www.eclipsepass.org",
```

Comment:
so no `www.eclipsepass.org`?

Reply:
I previously assumed there will be `www.*` as it is customary to have CNAME pointing www to '@' . But the issue doesn't actually call for it, but for `demo` and `nightly` instead. So I have removed it until it is explicitly requested.